blog/relacja-z-meetupu-hitachi-karate/index.md

```diff
@@ -0,0 +1,84 @@
+
+### Wszystko ma swoje priorytety
+
+![](images/andy_prez.jpg)
+
+Mimo opornego projektora, który przestał działać jak tylko do niego poszedłem (mam tę moc, mam tę moc!), udało mi się wyświetlić slajdy i opowiedzieć o zarządzaniu priorytetami projektów dokumentacyjnych. Podczas mojej blisko rocznej przygody w firmie Hitachi Energy miałem okazję przećwiczyć dostosowywanie priorytetów w dynamicznym środowisku, w którym tech writer ma więcej niż jeden projekt pod swoją pieczą. Cieszę się, że mogłem podzielić się obserwacjami, które mogą być pomocne również w kontekstach pozadokumentacyjnych, a także zaprezentowałem parę czerstwych memów własnego autorstwa.
+
+![](images/lee.png)
+
+Uczestnicy i uczestniczki zdarzenia okazywali zainteresowanie, zadali nawet pytania, a po zapaleniu świateł wszyscy byli obudzeni ;-) Dla mnie to definicja sukcesu.
+
+### Wejście Smoków
+
+![](images/1694696568105.jpg)
+
+Wisienką na torcie była rozmowa z sensei Mają Wójcik i sensei Jakubem Piaseckim, którzy opowiadali:
+* o dążeniu do mistrzostwa w dziedzinie karate Oyama
+* o pozytywnych wartościach sportowych jakie niesie ze sobą karate
+* o tym jak znajdować w sobie wewnętrzną siłę, by systematycznie rozwijać swoje umiejętności
+* o tym, jak przekonać siebie samego do pójścia na pierwszy trening
+
+Mieliśmy przyjemność posłuchać opowieści młodych, ambitnych trenerów, a ich rozważania na temat samodoskonalenia i sposobów radzenia sobie z trudnościami mają również zastosowanie w kontekście technical writingu. Ważnym przesłaniem tamtego wieczora było to, że uprawianie sportu jest dobrą metodą na zagwarantowanie sobie właściwego stanu mentalnego. Aktywność fizyczna jest świetnym remedium na pracę umysłową w świecie IT: pozwala na zachowanie wewnętrznej równowagi i wydobycie pełni swojego potencjału.
+
+![](images/sensei.jpg)
+
+Jakub wspomniał również o swojej drugiej pasji życiowej, którą jest fotografia. Karate było motywem przewodnim jego pracy dyplomowej, a przykłady jego prac mogliśmy podziwiać w trakcie spotkania.
+
+![](images/interview_karate2.png)
+
+Podziękowania dla [Piotra Sroki](https://www.linkedin.com/in/piotrsrokatechwriter/) i [Mateusza Barszcza](https://www.linkedin.com/in/mattbarszcz/), którzy przygotowali pytania i wcielili się role moderatorów spotkania.
+
+### Po prezentacjach
+![](images/team_he.png)
+Po długim i pełnym wrażeń wieczorze, gospodarze i goście mogli poczęstować się przekąskami, powymieniać się uwagami i poznać nowych ludzi. Każdy mógł zabrać do domu wybrany gadżet z logo Hitachi Energy. W tym miejscu podziękowania dla [Sandry Cieniawskiej](https://www.linkedin.com/in/sandracieniawska/) i [Klaudii Sucharzewskiej](https://www.linkedin.com/in/klaudia-sucharzewska-048ba721a/), która pomogły w zorganizowaniu kateringu i gadżetów.
+
+Słyszałem, że uczestnicy chwalili sobie to wydarzenie i odbiło się ono w firmie pewnym echem :)
```

Comment:
"odbiło się ono w firmie pewnym echem" - mam wrażenie, że to zdanie to otwarcie jakiejś anegdoty, ale brakuje tej anegdoty. Może chcesz napisać dlaczego masz takie wrażenie, na przykład ludzie cytowali fakty o Japonii przy kawie?

Reply:
nie chcialem zeby brzmialo to zbyt bunczucznie, np.: "to zdarzenie bylo na ustach wszystkich w firmie", kojarze jakies komentsy na intranecie i tez zaslyszane pozytywne opinie ludzi pozadokumentacyjnych. Nie brzmi to jak wyborna angedota :)
nie wiem jeszcze czym to zastapic, pomysle.

Reply:
Może też zostać jak jest, nie jest wcale źle 💜